### DISCUSSION TITLE
Reminder frequency could be set on weekly or monthly basis also.

### DISCUSSION THREAD

**Discussion by fsmangolini** [iSoron/uhabits#652] [Feature Requests]
So that if I have a habit to be done once or twice a month, I could set an alarm to remind me just then, not every day or week...

**Comment by sanjayvsingh** [comment]
This would be helpful. For example, I have a "Donate to charity" habit set up with a custom period to "Repeat 2 times in 30 days." It would be nice to have this set up to repeat "Every month."
